pages/_document.tsx

```diff
@@ -6,7 +6,7 @@ class MyDocument extends Document {
     const { hostname } = new URL(uri);
 
     return (
-      <Html lang="en" className="bg-gray-50">
+      <Html lang="en">
```

Comment:
Shouldn't language be taken from the Regions provider?

Reply:
Fixed:

```
// _document.tsx

class MyDocument extends Document<{ lang?: string }> {
  static async getInitialProps(ctx: DocumentContext) {
    const initialProps = await Document.getInitialProps(ctx);

    return { ...initialProps, lang: ctx?.query?.locale };
  }

  render() {
    const uri = process.env.NEXT_PUBLIC_API_URI!;
    const { hostname } = new URL(uri);

    return (
      <Html lang={this.props.lang}>
        <Head>
          <link rel="preconnect" href={`//${hostname}`} crossOrigin="true" />
          <link rel="dns-prefetch" href={`//${hostname}`} />
        </Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}

export default MyDocument;

```